ui/src/app/components/summary/summary.component.scss

```diff
@@ -2,6 +2,7 @@
   height: calc(100vh - 271px);
   position: relative;
   overflow-y: auto;
+  background-color: white;
```

Comment:
Add white background to rules component background as well.

Reply:
Added